engines/sci/engine/kgraphics.cpp

```diff
@@ -1253,6 +1253,33 @@ reg_t kShow(EngineState *s, int argc, reg_t *argv) {
 reg_t kRemapColors(EngineState *s, int argc, reg_t *argv) {
 	uint16 operation = argv[0].toUint16();
 
+	//Kawa says: the SCI16 source leak has SCI2-style RemapColors, but not all are used.
```

Comment:
Please remove this comment, this is concerning. And amend this commit, not introducing one on the top.

Reply:
I can't. Git on the command line is, let's say, aggressively secure, and GitHub Desktop doesn't seem to expose the required features to do either of these things. Best thing I can do is toss this pull request and try again from scratch.

Reply:
Well, this comment is simply a no go to our main repo.

Reply:
Have squashed commits and removed that comment...
The result is here as : https://github.com/digitall/scummvm/tree/dating_pool

You can replace this pull request with this by replacing your master branch with that branch's contents ... since you opened this PR from your master branch.

Reply:
It's not a matter of confidence, it's security. The git command line tool simply refuses to push.

Reply:
@Kawa-oneechan:
> The git command line tool simply refuses to push.

git push origin +branch_you_want_to_force_eg_master

Reply:
@Kawa-oneechan: Oh, sorry.
"**GitHub Desktop** doesn't seem to expose the required features"

That project's devs declined the [feature request](https://github.com/desktop/desktop/issues/3580)

Reply:
It's an SSL, TSL whatever _thing_. I stuck to HG for so long until I tried GitHub Desktop and found that actually works and lets me push things.

Reply:
@Kawa-oneechan:
> It's an SSL, TSL whatever *thing*.

Oh outdated TLS. [I ran into that](https://forums.scummvm.org/viewtopic.php?f=1&t=14739#p86494).

> The git that ships with that MSYS is so old that its HTTPS requests fail. I ran a separately installed git in a non-MSYS prompt.

Reply:
At this point I'd either just drop this pull request and start over to make it a single commit, or drop it and go with digitall's.

Reply:
@Kawa-oneechan : If you enable "Allow Edits from Maintainers", I can force-push my single commit version over this pull request..
https://blog.github.com/2016-09-07-improving-collaboration-with-forks/

Reply:
https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Reply:
My separate install is [Git for Windows](https://gitforwindows.org/) btw.

On further inspection, the MSYS I talked about on the forum did not ship with git. I must've confused it with an MSYS build environment I had on my hard drive left over from another project.

Reply:
Ah, you had already enabled the maintainer edit option, so have tried to force push this...